src/status_im/accounts/create/core.cljs

```diff
@@ -145,6 +109,157 @@
                               (dissoc :password :password-confirm :name :error)))}
             (navigation/navigate-to-cofx :create-account nil)))
 
+(fx/defn intro-wizard
+  {:events [:accounts.create.ui/intro-wizard]}
+  [{:keys [db] :as cofx}]
+  (fx/merge {:db (assoc db :intro-wizard {:step 1 :encrypt-with-password? true})}
+            (navigation/navigate-to-cofx :intro-wizard nil)))
+
+(fx/defn intro-step-back
+  {:events [:intro-wizard/step-back-pressed]}
+  [{:keys [db] :as cofx}]
+  (let  [step (get-in db [:intro-wizard :step])]
+    (if (< 1 step)
+      (fx/merge {:db (cond-> (assoc-in db [:intro-wizard :step] (dec step))
+                       (#{4 5} step)
+                       (assoc-in [:intro-wizard :key-code] nil)
+                       (= step 4)
+                       (assoc-in [:intro-wizard :encrypt-with-password?] false)
+                       (= step 5)
+                       (assoc-in [:intro-wizard :confirm-failure?] false))}
+                (navigation/navigate-to-cofx :intro-wizard nil))
+
+      (fx/merge {:db (dissoc db :intro-wizard)}
+                (navigation/navigate-to-clean :intro nil)))))
+
+(fx/defn intro-step-forward
+  {:events [:intro-wizard/step-forward-pressed]}
+  [{:keys [db] :as cofx} {:keys [skip?] :as opts}]
+  (let  [step (get-in db [:intro-wizard :step])]
+
+    (cond (= step 7)
```

Comment:
Out of scope but keywords would be more visible

Reply:
Agree, converted to keywords